site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,466 @@
+
+### Threaded and accelerated canvas rendering
+
+This is the project that put in place the architectural pieces that made OffscreenCanvas possible.
+
+<table class="width-full">
+  <thead>
+    <tr>
+      <td></td>
+      <th>2011</th>
+      <th>2014</th>
+      <th>2015</th>
+      <th>2016</th>
+      <th>2017</th>
+      <th>2018</th>
+      <th>2019</th>
+      <th>2020</th>
+      <th>2021</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>CompositeAfterPaint</th>
+      <td></td>
+      <td>Start</td>
+      <td>Ship display lists</td>
+      <td></td>
+      <td>Ship new invalidation</td>
+      <td>Ship property trees part 1</td>
+      <td>Ship property trees part 2</td>
+      <td></td>
+      <td>Ship SVG. Ship rest. Finish!</td>
+    </tr>
+    <tr>
+      <th>LayoutNG</th>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>Start</td>
+      <td></td>
+      <td></td>
+      <td>Ship block flow</td>
+      <td>Ship flex, editing</td>
+      <td>Ship everything else. Finish!</td>
+    </tr>
+    <tr>
+      <th>GPU Acelaration everywhere</th>
+      <td></td>
+      <td>Start!
+      <br>Canvas support?
+      <br>Shipped on opt-in content on Android.</td>
+      <td>Threaded GPU Rasterization</td>
+      <td>Ship on Mac</td>
+      <td>GPU is used on &gt;60% of Android page views.</td>
+      <td>Ship on Windows, CrOS, and Android Go</td>
+      <td></td>
+      <td>Ship remaining Android content. Finish!</td>
+      <td></td>
+    </tr>
+    <tr>
+      <th>Threaded scrolling, animation, and decode</th>
+      <td>Initial support for threaded scroll and animation</td>
+      <td></td>
+      <td>Layer squashing</td>
+      <td>Universal overflow scrolling</td>
+      <td>Image decodes on compositor thread</td>
+      <td>Image Animations on compositor thread</td>
+      <td></td>
+      <td>Always composite fixed-position</td>
+      <td>Percentage transform animations, SVG animations.</td>
+    </tr>
+    <tr>
+      <th>Viz</th>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>Start</td>
+      <td></td>
+      <td>OOP-R shipped on Android, Mac and Windows.</td>
+      <td>OOP-D shipped. OOP-R shipped everywhere (except Canvas). SkiaRenderer shipped on Linux.</td>
+      <td>SkiaRenderer shipped on Windows & Android. Vulkan shipped on Android.</td>
+      <td>SkiaRenderer shipped on Mac (& CrOS soon). Finish!</td>
+    </tr>
+    <tr>
+      <th>Threaded / accelerated canvas</th>
+      <td></td>
+      <td></td>
+      <td>Start</td>
+      <td></td>
+      <td></td>
+      <td>Ship OffscreenCanvas</td>
+      <td>ImageBitmapRenderingContext. Finish!</td>
+      <td></td>
+      <td>Ship OOPR. Finish!</td>
+    </tr>
+  </tbody>
+</table>
+
+### Definition of terms in the above chart
+
+OOP-D
+: Out of process display compositor.
+Display compositing is the same kind of activity as an OS compositor.
```

Comment:
You lost the hyperlink for "OS compositor" that's in the original document. Should link to:

https://en.wikipedia.org/wiki/Compositing_window_manager